Test errors in `tests/test_host_env.py` caused by psutil

This happens due to `psutil` having a soft requirement and apparently there's a breaking change somewhere in between
```
tests/test_host_env.py:26: in test_disk_info
    self.assert_disk_info()
tests/test_host_env.py:34: in assert_disk_info
    disk_info = host_env.get_disk_details()
agentops/host_env.py:113: in get_disk_details
    partitions = psutil.disk_partitions()
tests/test_host_env.py:23: in <lambda>
    @patch("psutil.disk_partitions", new=lambda: [mock_partitions()[0]])
tests/test_host_env.py:10: in mock_partitions
    sdiskpart(device="/dev/sda1", mountpoint="/", fstype="ext4", opts="rw,relatime"),
E   TypeError: sdiskpart.__new__() missing 2 required positional arguments: 'maxfile' and 'maxpath'
```

The is failing locally but not in CI because of different versions of `psutil`. In newer versions (5.9.0+), `sdiskpart` requires additional fields (`maxfile` and `maxpath`) that weren't needed in older versions.

--- a/tests/test_host_env.py
+++ b/tests/test_host_env.py
@@ -1,15 +1,27 @@
 from unittest.mock import patch
 from agentops import host_env
+import psutil
 
 # noinspection PyProtectedMember
 from psutil._common import sdiskpart, sdiskusage
 
 
 def mock_partitions():
-    return [
-        sdiskpart(device="/dev/sda1", mountpoint="/", fstype="ext4", opts="rw,relatime"),
-        sdiskpart(device="z:\\", mountpoint="z:\\", fstype="ntfs", opts="rw,relatime"),
-    ]
+    # Try to create with new fields first, fall back to old format if it fails
+    try:
+        return [
+            sdiskpart(  # noqa: E501
+                device="/dev/sda1",
+                mountpoint="/",
+                fstype="ext4",
+                opts="rw,relatime",
+                maxfile=255,  # type: ignore
+                maxpath=4096,  # type: ignore
+            )
+        ]
+    except TypeError:
+        # Fallback for older versions that don't have maxfile/maxpath
+        return [sdiskpart(device="/dev/sda1", mountpoint="/", fstype="ext4", opts="rw,relatime")]
 
 
 def mock_disk_usage(partition):

